.github/workflows/build-and-release.yml

```diff
@@ -238,11 +239,23 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Bump Version
+        id: bump-version
+        run: |
+          python tools/release/bump_version.py
+
       - uses: stefanzweifel/git-auto-commit-action@v5
         with:
-          file_pattern: '*/version.json'
```

Comment:
Should package.json be impacted too ?

Reply:
Is package.json a file that should be updated after releasing?

Reply:
yes it would be nice